include/camunda.mk

```diff
@@ -0,0 +1,86 @@
+.PHONY: camunda
```

Comment:
I'd suggest to rename the dir to `make` for clarity.
If you plan to add more includes, then it should be `include/make`.

Reply:
I tried to address you feedback with a README file in the folder: https://github.com/falko/camunda-8-helm-profiles/tree/master/include#readme